static/app/components/devtoolbar/components/teams/useTeams.tsx

```diff
@@ -0,0 +1,30 @@
+import {useMemo} from 'react';
+
+import type {Team} from 'sentry/types/organization';
+
+import useConfiguration from '../../hooks/useConfiguration';
+import useFetchApiData from '../../hooks/useFetchApiData';
+
+interface Props {
+  idOrSlug?: string;
+}
+
+export default function useTeams({idOrSlug}: Props, opts?: {enabled: boolean}) {
+  const {organizationSlug} = useConfiguration();
+
+  return useFetchApiData<Team[]>({
+    queryKey: useMemo(
+      () => [
+        `/organizations/${organizationSlug}/teams/`,
```

Comment:
can we replace `organizationSlug` here with `idOrSlug`? (do we need both?)

Reply:
`organizationSlug` could be a slug or id, but it's different from `idOrSlug` in this same file.. 

`idOrSlug` is meant to be the teamId or teamSlug. I tried to make the method signature read well: ie `useTeams({idOrSlug: 'replay-team'})` but maybe it's more confusing inside the method because there are multiple slugs available.